package.json

```diff
@@ -104,6 +105,7 @@
     "husky": "^4.3.0",
     "ignore-loader": "^0.1.2",
     "jest": "^26.4.2",
+    "json-bigint": "^1.0.0",
```

Comment:
I have used the library mentioned in the issue to make the test more accurate

Reply:
That’s okay. We can also leverage examples repo (smoke tests) for purposes like this. I didn’t decide yet where the border lies on this.

Reply:
ok, fine